packages/react-core/src/components/Checkbox/Checkbox.tsx

```diff
@@ -10,8 +10,12 @@ export interface CheckboxProps
     OUIAProps {
   /** Additional classes added to the checkbox. */
   className?: string;
-  /** Additional classed added to the radio input */
+  /** Additional classed added to the checkbox input. */
   inputClassName?: string;
+  /** Flag to show if the checkbox label is wrapped on small screen. Will only apply if component prop is not specified. */
+  isLabelWrapped?: boolean;
+  /** Flag to show if the checkbox label is shown before the checkbox button. */
+  isLabelBeforeButton?: boolean;
```

Comment:
This is fine but I wonder if in a breaking change we should rename this and the radio to remove the "button" part and use something more generic. `isLabelBeforeInput` would probably be more accurate, but it could just be `isLabelStart` or something along those lines - do we have any kind of pattern for naming props like this? This prop is a boolean, but the button and accordion have props `iconPosition="[start/end]"` and `togglePosition=[start/end]` that do something similar.

Reply:
I checked the core docs and it has a "Reversed" example, so possibly `isReversed` or `isLabelReversed`?

<img width="368" alt="Screenshot 2024-01-09 at 11 11 12" src="https://github.com/patternfly/patternfly-react/assets/84135613/8425ccbd-08dd-445e-89b5-4e3bad83137f">

I wonder whether we rename it now to have less codemods, or just keep it consistent in v5 so we keep the `isLabelBeforeButton` to match the Radio prop.